lib/Expectation.lua

```diff
@@ -197,6 +197,33 @@ function Expectation:equal(otherValue)
 	return self
 end
 
+--[[
+	Assert that our expectation value is equal to another value within some limit, inclusive
+]]
+function Expectation:near(otherValue, limit)
+	local result = (math.abs(self.value - otherValue) <= limit) == self.successCondition
+
+	local message = formatMessage(self.successCondition,
+		("Expected value %q (%s) within %q , got %q (%s) instead"):format(
```

Comment:
This message seems kind of clunky.

I imagine our message should be something like

> Expected value to be within %f of %f, but got %f instead